nengo_loihi/tests/test_interneurons.py

```diff
@@ -0,0 +1,72 @@
+import nengo
+import numpy as np
+import pytest
+
+from nengo_loihi.builder import (
+    Model, NoisyInterneurons, OnOffInterneurons,
+    Preset5Interneurons, Preset10Interneurons,
+)
+
+
+@pytest.mark.parametrize('interneurons, tolerance', [
+    (OnOffInterneurons(), 0.32),
+    (NoisyInterneurons(5), 0.11),
+    (NoisyInterneurons(10), 0.10),
+    (Preset5Interneurons(), 0.03),
+    (Preset10Interneurons(), 0.03),
+])
+def test_add_inputs(interneurons, tolerance, Simulator, seed, plt):
+    sim_time = 2.
+    pres_time = 0.5
+    eval_time = 0.25
+
+    stim_fn_a = nengo.processes.Piecewise({
+        t: (2*(i % 2) - 1) * 0.4
+        for i, t in enumerate(np.arange(0, sim_time, pres_time))})
+    stim_fn_b = stim_fn_a
```

Comment:
Why is this a new variable? This could just be `stim_fn` and passed to both nodes?

Reply:
I've actually changed the test to have different functions for each node now, so it's more discerning.